crates/astria-sequencer-relayer/src/types.rs

```diff
@@ -1,200 +1,281 @@
-use std::convert::TryFrom;
+use std::{
```

Comment:
this file is basically sequencer_block.rs, i renamed it since the previous types.rs was deleted